test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -0,0 +1,696 @@
+#ifndef PSQLODBC_TESTS_COMMON_H
+#define PSQLODBC_TESTS_COMMON_H
+
+#include <string>
+#include <vector>
+
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "../src/drivers.h"
+#include <gtest/gtest.h>
+#include <sqlext.h>
+
+using std::vector;
+using std::string;
+using std::pair;
+
+string InitializeInsertString(const vector<string> &insertedValues);
+
+void testCommonColumnAttributes(string tableName, vector<pair<string, string>> tableColumns, string orderByColumnName, vector<int> lengthExpected, vector<int> precisionExpected, vector<int> scaleExpected, vector<string> nameExpected);
```

Comment:
For headers that are this long, could you introduce line breaks into them so it can fit without having to scroll?

Reply:
Done.